perfmetrics/scripts/run_load_test_and_fetch_metrics.sh

```diff
@@ -2,6 +2,9 @@
 set -e
 echo Running fio test..
 fio job_files/seq_rand_read_write.fio --lat_percentiles 1 --output-format=json --output='output.json'
+echo Logging fio results
+cp output.json gcs/Fio-logs/output-$(date '+%Y-%m-%d').json
```

Comment:
lets follow small case for all directories/buckets which we create. rename to fio-logs

Reply:
Done